tensorflow/core/util/mkl_types.h

```diff
@@ -112,6 +112,7 @@ namespace tensorflow {
 #define TENSOR_FORMAT_NHWC MKL_TENSOR_FORMAT_NHWC
 #define TENSOR_MAX_DIMS MKLDNN_MAX_NDIMS
 #define GET_USR_MEM_PRIM_DESC(src) src.GetUsrMemDesc()
+#define BN_FLAGS mkldnn::batch_normalization_flags
```

Comment:
Continuing the discussion here since the old comment is outdated. You said
> mkldnn::batch_normalization_flags is in V1.x

Could you please point me to where the`batch_normalization_flags` class/struct is defined? I couldn't find it in v1.0, only found the `normalization_flags` class.